src/main/java/de/recipe/service/RecipeService.java

```diff
@@ -0,0 +1,6 @@
+package de.recipe.service;
+
+public interface RecipeService {
+
+
```

Comment:
please put all methods into interface also. otherwise you won't be able to autowire it